src/interface/report/PATCHES.ts

```diff
@@ -5,67 +5,97 @@
   Timestamps are for the NA Region to match WCL
 */
 
+import Expansion from 'game/Expansion';
+
 export interface Patch {
   name: string;
   timestamp: number;
   urlPrefix: string;
   isCurrent: boolean;
+  gameVersion: number;
+  expansion: Expansion;
 }
 
 const PATCHES: Patch[] = [
+  {
+    name: '3.4.0',
+    timestamp: 1664229600000, // GMT: Monday, 26 September 2022 22:00:00
+    urlPrefix: '',
+    isCurrent: true,
+    gameVersion: 4, // WotLK
```

Comment:
I was thinking if we could turn this into an enum? It occours several times now through out more of the code base. Probably not something to change in the current PR but somethig to put on the backlog?

Reply:
Probably a good thing for a follow on PR